tests/security/apparmor/aa_logprof.pm

```diff
@@ -58,16 +59,25 @@ sub run {
 
     validate_script_output "aa-complain -d $aa_tmp_prof usr.sbin.nscd", sub { m/Setting.*complain/ };
 
+    # For tumbleweed, unload /usr/sbin/nscd profile in case, clean up the audit.log
+    if (is_tumbleweed) {
+        script_run "echo '/usr/sbin/nscd {}' | apparmor_parser -R";
+    }
```

Comment:
Maybe you could do it for all situations, not only the tumbleweed. Although it is not failed in SLE currently, but what if in future apparmor version on SLE bumped to the current version?

Reply:
Thanks for the review, 
as discussed offline it is hard to do for all situations I would like to keep the code.